components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -254,13 +267,15 @@ subroutine linv3_strat_chem_solve(ncol, lchnk, xvmr, h2ovmr, xsfc, o3col, temp,
     real(r8), intent(inout), dimension(ncol ,pver,gas_pcnst) :: xvmr      ! volume mixing ratio for all
     real(r8), intent(in)   , dimension(ncol ,pver) :: h2ovmr              ! h2o vapor volume mixing ratio 
     real(r8), intent(inout)                        :: xsfc(4,ncol)        ! surface o3,n2o,noy,ch4 from linoz table
-    real(r8), intent(in)   , dimension(ncol ,pver) :: o3col               ! ozone column above box (mol/cm^2)
+    real(r8), intent(in)   , dimension(ncol ,pver) :: o3col               ! Linoz ozone column above box (mol/cm^2)
```

Comment:
This comment is a bit misleading. How about "! ozone column above box, can be either "O3" column or "O3LNZ" column"?

Reply:
Changed.